packages/adminpanel-v2/src/stores/CustomerStore.ts

```diff
@@ -0,0 +1,84 @@
+//import { TCustomer, TProject, TActivity} from "$lib/types"
+import type { TCustomer } from "$lib/types";
+import { get, writable } from "svelte/store";
+import { customers as mock}  from "$lib/mock/customers";
+
+type TCustomerStore = {
+	customers: TCustomer[],
+	active: {
+		customer: number | undefined,
+		project: number | undefined,
+		activity: number | undefined
+	}
+}
+
+function createCustomers() {
+	const customerStore = writable<TCustomerStore>(
+		{
+			customers: mock,
+			active: {
+				customer: undefined,
+				project: undefined,
+				activity: undefined
+			}
+		}
+	);
+	const { subscribe, set, update } = customerStore;
+
+	const fetchCustomers = () => {
+		//fetch from backend
+	}
+
+	const setCustomers = () => {
+		update((n) => {n.customers = mock; return n})
+		
+	}
+
+    const setCustomer = (id: number) => {
```

Comment:
Is this the customer_id (kunde_id) that Marcus talked about, or is this just a random generated ID number we use to have uniuqe customer IDs?

Reply:
The id here refers to the id defined in the customer type used in the mock data. Once the store is connected to the api, the object structure will have to be updated to reflect the data fetched from the backend.

Reply:
Thank you! 🥇